snippets/java/diamondSquare.md

```diff
@@ -0,0 +1,65 @@
+# Print a Diamond pattern enclosed in a square
```

Comment:
Remove this file from this pull request. Wait, didn't you make separate branches for separate issues? If not then kindly do that else clean up the extra file in this  pull request

Reply:
Ok will do it .Thanks for the review.